app/main.py

```diff
@@ -1,6 +1,16 @@
 from typing import Callable
 
+cache_dict = {}
+
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    def inner(*args, **kwargs) -> int:
+        global cache_dict
+        if (func, args) in cache_dict:
+            print("Getting from cache")
+            return cache_dict[(func, args)]
+        else:
```

Comment:
There is no need for `else` since the code will not execute after the `return` statement anyway. Please, remove the redundant `else`